tutorials/ui/size_and_anchors.rst

```diff
@@ -17,37 +15,53 @@ and mobile phones have different resolutions and aspect ratios.
 There are several ways to handle this, but for now, let's just imagine
 that the screen resolution has changed and the controls need to be
 re-positioned. Some will need to follow the bottom of the screen, others
-the top of the screen, or maybe the right or left margins.
+the top of the screen, or maybe the right or left borders.
+
+.. image:: img/ui_anchor_and_margins.webp
+
+This is done using the *margin* and *anchor* properties. Margins represent a
+distance in pixels relative to the top-left corner of the parent control or
+(in case there is no parent control) the viewport. Anchors adjust where the
+margin distances are relative *to*. We can think of them as a percentage of the
+parent's size, represented by a value between 0 and 1.0.
```

Comment:
Anchors should probably be explained before offsets. Also offsets are not relative to the top-left corner, that's only true by default. They are relative to anchor points, and anchor points in turn default to the top-left corner. I think this explanation here needs more work.

Reply:
Also you should be consistent when formatting numbers. It should be `0.0` throughout the article.